src/config/loadGitHubOrgsFromEnvironment.ts

```diff
@@ -1,53 +1,33 @@
-import {
-  AppAuthStrategyOptions,
-  GitHubIssuesSomeoneElseCaresAbout,
-} from '@/types';
+import { GitHubOrgConfig } from '@/types';
 
-// Config - loosely typed and ephemeral, used for collecting values found in
+// Don't use '@api' here, because it's ... not loaded yet, or something?
```

Comment:
😮 What error is being thrown here?

Reply:
```
$ yarn test -- src/config/loadGitHubOrgsFromEnvironment.test.ts
Error: Jest: Got error running globalSetup - /Users/chadwhitacre/workbench/getsentry/eng-pipes/test/jest.glo
balSetup.ts, reason: Your application tried to access @api/github, but it isn't declared in your dependencie
s; this makes the require call ambiguous and unsound.
  
Required package: @api/github (via "@api/github/org")
Required by: /Users/chadwhitacre/workbench/getsentry/eng-pipes/src/config/

Require stack:
- /Users/chadwhitacre/workbench/getsentry/eng-pipes/src/config/loadGitHubOrgsFromEnvironment.ts
- /Users/chadwhitacre/workbench/getsentry/eng-pipes/src/config/index.ts
- /Users/chadwhitacre/workbench/getsentry/eng-pipes/src/knexfile.ts
- /Users/chadwhitacre/workbench/getsentry/eng-pipes/test/jest.globalSetup.ts
- 
```

Reply:
```diff
diff --git a/src/config/loadGitHubOrgsFromEnvironment.ts b/src/config/loadGitHubOrgsFromEnvironment.ts
index 6a51857..6014d26 100644
--- a/src/config/loadGitHubOrgsFromEnvironment.ts
+++ b/src/config/loadGitHubOrgsFromEnvironment.ts
@@ -1,7 +1,7 @@
 import { GitHubOrgConfig } from '@/types';
 
 // Don't use '@api' here, because it's ... not loaded yet, or something?
-import { GitHubOrg } from '../api/github/org';
+import { GitHubOrg } from '@api/github/org';
 
 // Configs - loosely typed and ephemeral, used for collecting values found in
 // the environment. We check for missing values using this but not for types,
```